
# Overview

CardiacG2P is an evidence-based dataset for inherited cardiac condition
gene disease pairs. Each entry annotates a disease-associated gene with
information about the gene-disease relationship, including the
inheritance mode and allelic requirement, information pertaining to
disease mechanism (represented as a disease-associated variant
consequence), and known disease-relevant variant classes at a defined
locus.

This dataset is also available through G2P
<https://www.ebi.ac.uk/gene2phenotype>

## Citation

*Josephs, K.S., Roberts, A.M., Theotokis, P. et al. Beyond gene-disease
validity: capturing structured data on inheritance, allelic requirement,
disease-relevant variant classes, and disease mechanism for inherited
cardiac conditions. Genome Med 15, 86 (2023).*
<https://doi.org/10.1186/s13073-023-01246-8>

## Brugada Syndrome (BrS)

### Brugada Syndrome

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

## Other potentially relevant variant classes

Which variant classes should I filter for? The main table shows variant
classes that have been reported to cause disease. We would expect other
variant classes with similar consequences to cause disease (e.g. if
NMD_triggering frameshift cause disease, then NMD_triggering nonsense
probably do too). This table below shows the full list of variant
classes mapped to each variant consequence.

<table class="table table-striped table-condensed" style="margin-left: auto; margin-right: auto;border-bottom: 0;">
<thead>
<tr>
<th style="border-bottom:hidden;padding-bottom:0; padding-left:3px;padding-right:3px;text-align: center; " colspan="3">

<div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">

Disease-associated variant consequence

</div>

</th>
</tr>
<tr>
<th style="text-align:left;">
Decreased_gene_product_level
</th>
<th style="text-align:left;">
Absent_gene_product_level
</th>
<th style="text-align:left;">
Altered_gene_product_sequence
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
splice_region_variant
</td>
<td style="text-align:left;">
splice_acceptor_variant
</td>
<td style="text-align:left;">
splice_acceptor_variant
</td>
</tr>
<tr>
<td style="text-align:left;">
splice_acceptor_variant
</td>
<td style="text-align:left;">
splice_acceptor_variant_NMD_triggering
</td>
<td style="text-align:left;">
splice_acceptor_variant_NMD_escaping
</td>
</tr>
<tr>
<td style="text-align:left;">
splice_acceptor_variant_NMD_triggering
</td>
<td style="text-align:left;">
splice_donor_variant
</td>
<td style="text-align:left;">
splice_donor_variant
</td>
</tr>
<tr>
<td style="text-align:left;">
splice_donor_variant
</td>
<td style="text-align:left;">
splice_donor_variant_NMD_triggering
</td>
<td style="text-align:left;">
splice_donor_variant_NMD_escaping
</td>
</tr>
<tr>
<td style="text-align:left;">
splice_donor_variant_NMD_triggering
</td>
<td style="text-align:left;">
start_lost
</td>
<td style="text-align:left;">
frameshift_variant_NMD_escaping
</td>
</tr>
<tr>
<td style="text-align:left;">
start_lost
</td>
<td style="text-align:left;">
frameshift_variant
</td>
<td style="text-align:left;">
stop_gained_NMD_escaping
</td>
</tr>
<tr>
<td style="text-align:left;">
frameshift_variant
</td>
<td style="text-align:left;">
frameshift_variant_NMD_triggering
</td>
<td style="text-align:left;">
stop_lost
</td>
</tr>
<tr>
<td style="text-align:left;">
frameshift_variant_NMD_triggering
</td>
<td style="text-align:left;">
stop_gained
</td>
<td style="text-align:left;">
missense_variant
</td>
</tr>
<tr>
<td style="text-align:left;">
stop_gained
</td>
<td style="text-align:left;">
stop_gained_NMD_triggering
</td>
<td style="text-align:left;">
inframe_insertion
</td>
</tr>
<tr>
<td style="text-align:left;">
stop_gained_NMD_triggering
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
inframe_deletion
</td>
</tr>
</tbody>
<tfoot>
<tr>
<td style="padding: 0; " colspan="100%">
<span style="font-style: italic;text-decoration: underline;">Note:
</span>
</td>
</tr>
<tr>
<td style="padding: 0; " colspan="100%">
<sup></sup> Variant classes with a likelihood score of ‘3:possible’,
‘4:probable’ and ‘5:almost always’ <br>For the different disease
associated variant consequences, see Tables 2 and 3 from Roberts et al.
</td>
</tr>
</tfoot>
</table>

*Roberts AM, DiStefano MT, Riggs ER, Josephs KS, Alkuraya FS, Amberger
J, Amin M, Berg JS, Cunningham F, Eilbeck K, Firth HV, Foreman J, Hamosh
A, Hay E, Leigh S, Martin CL, McDonagh EM, Perrett D, Ramos EM, Robinson
PN, Rath A, Sant DW, Stark Z, Whiffin N, Rehm HL, Ware JS. Toward robust
clinical genome interpretation: Developing a consistent terminology to
characterize Mendelian disease-gene relationships-allelic requirement,
inheritance modes, and disease mechanisms. Genet Med. 2024
Feb;26(2):101029. doi: 10.1016/j.gim.2023.101029. Epub 2023 Nov 17.
PMID: 37982373; PMCID: PMC11039201;*
<https://doi.org/10.1016/j.gim.2023.101029>
