Cardiac Gene2Phenotype Gene Panel
================

## Overview

CardiacG2P is an evidence-based dataset for inherited cardiac condition
gene disease pairs. Each entry annotates a disease-associated gene with
information about the gene-disease relationship, including the
inheritance mode and allelic requirement, information pertaining to
disease mechanism (represented as a disease-associated variant
consequence), and known disease-relevant variant classes at a defined
locus.

This dataset is also available through G2P
<https://www.ebi.ac.uk/gene2phenotype>

## Citation

*Josephs, K.S., Roberts, A.M., Theotokis, P. et al. Beyond gene-disease
validity: capturing structured data on inheritance, allelic requirement,
disease-relevant variant classes, and disease mechanism for inherited
cardiac conditions. Genome Med 15, 86 (2023).*
<https://doi.org/10.1186/s13073-023-01246-8>

## Inherited Arrhythmia Syndromes

### Brugada Syndrome (BrS)

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

##### SCN5A

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
PMIDs
</td>
<td style="text-align:left;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
HGNC ID
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
G2P gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br><br> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br><br> SCN5A-related Brugada syndrome is
characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br><br>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br><br> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

## Other potentially relevant variant classes

Which variant classes should I filter for? The main table shows variant
classes that have been reported to cause disease. We would expect other
variant classes with similar consequences to cause disease (e.g. if
NMD_triggering frameshift cause disease, then NMD_triggering nonsense
probably do too). This table below shows the full list of variant
classes mapped to each variant consequence.

<table class="table table-striped table-condensed" style="margin-left: auto; margin-right: auto;border-bottom: 0;">
<thead>
<tr>
<th style="border-bottom:hidden;padding-bottom:0; padding-left:3px;padding-right:3px;text-align: center; " colspan="3">

<div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">

Disease-associated variant consequence

</div>

</th>
</tr>
<tr>
<th style="text-align:left;">
Decreased_gene_product_level
</th>
<th style="text-align:left;">
Absent_gene_product_level
</th>
<th style="text-align:left;">
Altered_gene_product_sequence
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
splice_region_variant
</td>
<td style="text-align:left;">
splice_acceptor_variant
</td>
<td style="text-align:left;">
splice_acceptor_variant
</td>
</tr>
<tr>
<td style="text-align:left;">
splice_acceptor_variant
</td>
<td style="text-align:left;">
splice_acceptor_variant_NMD_triggering
</td>
<td style="text-align:left;">
splice_acceptor_variant_NMD_escaping
</td>
</tr>
<tr>
<td style="text-align:left;">
splice_acceptor_variant_NMD_triggering
</td>
<td style="text-align:left;">
splice_donor_variant
</td>
<td style="text-align:left;">
splice_donor_variant
</td>
</tr>
<tr>
<td style="text-align:left;">
splice_donor_variant
</td>
<td style="text-align:left;">
splice_donor_variant_NMD_triggering
</td>
<td style="text-align:left;">
splice_donor_variant_NMD_escaping
</td>
</tr>
<tr>
<td style="text-align:left;">
splice_donor_variant_NMD_triggering
</td>
<td style="text-align:left;">
start_lost
</td>
<td style="text-align:left;">
frameshift_variant_NMD_escaping
</td>
</tr>
<tr>
<td style="text-align:left;">
start_lost
</td>
<td style="text-align:left;">
frameshift_variant
</td>
<td style="text-align:left;">
stop_gained_NMD_escaping
</td>
</tr>
<tr>
<td style="text-align:left;">
frameshift_variant
</td>
<td style="text-align:left;">
frameshift_variant_NMD_triggering
</td>
<td style="text-align:left;">
stop_lost
</td>
</tr>
<tr>
<td style="text-align:left;">
frameshift_variant_NMD_triggering
</td>
<td style="text-align:left;">
stop_gained
</td>
<td style="text-align:left;">
missense_variant
</td>
</tr>
<tr>
<td style="text-align:left;">
stop_gained
</td>
<td style="text-align:left;">
stop_gained_NMD_triggering
</td>
<td style="text-align:left;">
inframe_insertion
</td>
</tr>
<tr>
<td style="text-align:left;">
stop_gained_NMD_triggering
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
inframe_deletion
</td>
</tr>
</tbody>
<tfoot>
<tr>
<td style="padding: 0; " colspan="100%">
<span style="font-style: italic;text-decoration: underline;">Note:
</span>
</td>
</tr>
<tr>
<td style="padding: 0; " colspan="100%">
<sup></sup> Variant classes with a likelihood score of ‘3:possible’,
‘4:probable’ and ‘5:almost always’ for the different disease associated
variant consequences, see Tables 2 and 3 from Roberts et al.
</td>
</tr>
</tfoot>
</table>

*Roberts AM, DiStefano MT, Rooney Riggs E, Josephs KS, Alkuraya FS,
Amberger J, et al. Towards robust clinical genome interpretation:
developing a consistent terminology to characterize disease-gene
relationships - allelic requirement, inheritance modes and disease
mechanisms. MedRxiv. 2023;*
<https://doi.org/10.1101/2023.03.30.23287948>
